docs/community_plugins.md

```diff
@@ -3,6 +3,47 @@
 Share your own plugins with everyone by adding to the list at [vatz-community-plugins](https://github.com/dsrvlabs/vatz-community-plugins) <br>
 Other contribution policies are available at [contribution guide](https://github.com/dsrvlabs/vatz/blob/main/docs/contributing.md). 
 
-<a href="https://github.com/dsrvlabs/vatz-community-plugins">
-	<img alt="Qries" src="https://github.com/dsrvlabs/vatz/assets/6308023/22c1a2cd-8abb-4ce6-813b-c173704ac80e">
-</a>
\ No newline at end of file
+---
+
+## VATZ Community Plugins
+
+`vatz-community-plugins` is a collection of unofficial plugins contributed by members of the community who wish to extend the functionality of the VATZ project with their own plugins.
+- This repository hosts all unofficial plugins for any protocol.
+- You are welcome to share your own plugins with the broader community.
+- Please note that community plugins are not managed by DSRV Labs. (Only official VATZ plugins are managed by DSRV.)
```

Comment:
DSRV Labs -> DSRV

Reply:
`(Only official VATZ plugins are managed by DSRV.)` This sentence seems redundant because we'll mention it below.

Reply:
Skip to fix it 
refer to https://github.com/dsrvlabs/vatz/pull/600#discussion_r1794288375